untitled/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.6.2
```

Comment:
You can remove the 'untitled' subdirectory.

Reply:
Dirrectory removed

Reply:
Confirmed.